client/src/Components/VideoCard.js

```diff
@@ -0,0 +1,48 @@
+import { useState } from "react";
+
+const VideoCard = ({ video, removeVideo }) => {
+  const [rating, setRating] = useState(video.rating);
```

Comment:
It looks like these ratings aren't stored in the database, so if you refresh the page they'll go away, and if multiple users look at the page they won't see each other's ratings. How would you add these to the database so that they're persisted?